src/index.html

```diff
@@ -1,12 +1,74 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
+<html lang="en" class="page">
+  <head >
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="calendar__day calendar__day--number--1"></div>
+
+      <div class="calendar__day calendar__day--number--2"></div>
+
+      <div class="calendar__day calendar__day--number--3"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```
use nth child or nth of type in css